compiler/src/dotty/tools/dotc/core/tasty/TreeUnpickler.scala

```diff
@@ -1165,6 +1168,16 @@ class TreeUnpickler(reader: TastyReader,
             case SELECTouter =>
               val levels = readNat()
               readTerm().outerSelect(levels, SkolemType(readType()))
+            case SELECTin =>
+              var sname = readName()
+              val qual = readTerm()
+              val owner = readType()
+              val prefix = qual.tpe.widenIfUnstable
```

Comment:
To match what's done in accessibleDenot and typedSelect that probably needs to be:
```suggestion
              val prefix = maybeSkolemizePrefix(qual.tpe.widenIfUnstable, name)
```
(requires extracting the non-signed name from sname)

Reply:
This comment still needs to be addressed.